tests/testsuite/new.rs

```diff
@@ -75,7 +75,7 @@ fn both_lib_and_bin() {
 
 #[test]
 fn simple_git() {
-    cargo_process("new --lib foo").env("USER", "foo").run();
+    cargo_process("new --lib foo --edition 2015").env("USER", "foo").run();
```

Comment:
Is it correct to assume these are temporary? I imagine in theory at least we'd want to not have this here -- most users shouldn't be passing/working in 2015 edition locally I imagine. Or, at least, we'd want tests for both.

Anyway, not a concern that needs to be resolved in this PR I think.